api/models/MemberRole.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable camelcase */
+
+module.exports = bookshelf.Model.extend({
+  tableName: 'members_roles',
+  requireFetch: false,
```

Comment:
realizing that I think it would be good to add created_at and updated_at to this table to track when roles get assigned or changed

Reply:
thought of it here because its missing the timestamps property

Reply:
Same for GroupRole ?

Reply:
Doesn’t feel as important but I guess if you are adding then why not